### ISSUE TITLE
Suggestion: add an additional "Add Key" button on the left of each row to add a key at the start of each row

### ISSUE THREAD

**Issue by kennethso168** [Lixquid/app-unexpected-keyboard-layout-editor#16]
![image](https://github.com/Lixquid/app-unexpected-keyboard-layout-editor/assets/5262487/c543d43d-c678-4457-ab79-15ba85022cc6)

I'd like to add a key at the beginning of a row. However, currently keys can only be added at the end, and it is much more convenient to copy and paste the xml directly instead of using this editor.

It would be great if keys can be added to the start of the row using the editor too. I suggest adding an additional "Add Key" button on the left to achieve this.

**Comment by Lixquid** [comment]
I agree that this is an issue, but I'm worried about how much horizontal real estate would be sacrificed with an extra left column.

Another request that would hopefully also solve this issue would be a way to move keys and rows, but I'm not currently sure how best to implement this. Current working idea is a mode you can enter from the key dialog to allow for moving keys around.
